src/i18n/messages/es.messages.json

```diff
@@ -13,11 +13,14 @@
       "title": "Historial"
     },
     "nextQueries": {
-      "title": "Para ti"
+      "title": "Quizás te interese"
     },
     "nextQueriesGroup": {
-      "title": "Te podría interesar",
-      "message": "Esto es lo que otros compradores han buscado después de <span class=\"x-text x-text--bold\">\"{query}\"</span>"
+      "title": "Quizás te interese",
+      "message": "Los que buscaron <span class=\"x-text x-text--bold\">\"{query}\"</span> también han visto:"
+    },
+    "identifierResults": {
```

Comment:
Remove the idenftifierResults translation

Reply:
done